portal_recaptcha/migrations/12.0.1.0.1/pre-migration.py

```diff
@@ -0,0 +1,35 @@
+def migrate(cr, version):
```

Comment:
Can you remove the migration scripts for older versions (eg not present on the OCA ?) ?